src/main/java/net/openhft/chronicle/core/LicenceCheck.java

```diff
@@ -67,7 +67,7 @@ static void licenceExpiry(String product, Class<?> caller, BiConsumer<Long, Stri
             String expiryDateFile = product + ".expiry-date";
             try {
                 String source = new String(IOTools.readFile(LicenceCheck.class, expiryDateFile));
-                LocalDate expiryDate = LocalDate.parse(source);
+                LocalDate expiryDate = LocalDate.parse(source.substring(0, 10));
```

Comment:
Should we just do `source.trim()` that would presumably cover off the case where there is whitespace before the date too?

Reply:
yes good idea, and cleaner code too